youtube/MITx.yaml

```diff
@@ -0,0 +1,5 @@
+---
+offered_by: OCW
+channel_id: UCTBMWu8yshnAmpzR3MoJFtw
```

Comment:
it's a cosmetic thing but can we have 'channel_id' as the first metadata  ?

Reply:
sure